charts/barman/templates/configmap-barman-backups.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "barman.fullname" . }}-backups-config
+  labels:
+    {{- include "barman.labels" . | nindent 4 }}
+data:
+  {{- range .Values.barman.backups }}
+  {{ .postgresql.host }}.conf: |
+      [{{ .postgresql.host }}]
+      active = true
+      description = "PostgreSQL Database (Streaming-Only)"
```

Comment:
Do we want to make this description more descriptive by adding context that helps differentiate the individual backups in the config? TBH it's not like I know where the descriptions is used.

Reply:
The description is displayed when doing a `barman list-server`. Don't know either where else it is used. 
I don't think it's really needed to make this configurable since this environment doesn't support any other setups than "Streaming-Only". The server name should be enough to differentiate between the individual backups.

Reply:
It might also get used by barman web if we ever decide to deploy that.